final_task/rss_reader/News.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass
+import datetime
+import logging
+
+module_logger = logging.getLogger("rss_reader.scripts.News")
```

Comment:
Global variables are ususally declared in uppercase

Reply:
Done